dowhy/causal_refuters/dummy_outcome_refuter.py

```diff
@@ -19,6 +19,10 @@ class DummyOutcomeRefuter(CausalRefuter):
     - 'random_state': int, RandomState, None by default
     The seed value to be added if we wish to repeat the same random behavior. If we want to repeat the
     same behavior we push the same seed in the psuedo-random generator
+    - 'outcome_prediction_function': function pd.Dataframe -> np.ndarray, None

```

Comment:
simpler to just call it "outcome_function". We are not really doing any prediction per se, more closer to estimation.

Reply:
Yeah, makes sense.